compiler/src/iree/compiler/Utils/ElementPackingUtils.cpp

```diff
@@ -67,8 +67,8 @@ Value calculateStorageElementCountInBytes(Location loc,
   SmallVector<int64_t> paddedShape(shapedType.getShape());
   SmallVector<Value> paddedDynamicDims(dynamicDims.begin(), dynamicDims.end());
   auto encoding = IREE::Encoding::getEncodingAttr(shapedType);
-  if (encoding && !encoding.getRoundDimsToArray().empty()) {
-    auto roundDimsTo = encoding.getRoundDimsToArray();
+  if (encoding && !encoding.getMaxPadding()) {
+    auto roundDimsTo = encoding.getMaxPadding().getInt();
```

Comment:
I think the CI failures you're getting are because you need to drop the boolean negation (`!`) in this `if` condition:

```c++
if (encoding && !encoding.getMaxPadding()) {
//              ^ drop this `!`
```

Reply:
nice find! This is definitely my fault.